ui/components/app/advanced-gas-fee-popover/advanced-gas-fee-defaults/advanced-gas-fee-defaults.js

```diff
@@ -0,0 +1,76 @@
+import React, { useCallback, useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import Box from '../../../ui/box';
+import Typography from '../../../ui/typography';
+import CheckBox from '../../../ui/check-box';
+import I18nValue from '../../../ui/i18n-value';
+import {
+  COLORS,
+  DISPLAY,
+  FLEX_DIRECTION,
+  TYPOGRAPHY,
+} from '../../../../helpers/constants/design-system';
+import { useAdvanceGasFeePopoverContext } from '../context';
+import {
+  getIsAdvancedGasFeeDefault,
+  getAdvancedGasFeeValues,
+} from '../../../../selectors';
+import { setAdvancedGasFee } from '../../../../store/actions';
+
+const AdvancedGasFeeDefaults = () => {
+  const dispatch = useDispatch();
+
+  const {
+    baseFeeMultiplier,
+    maxPriorityFeePerGas,
+  } = useAdvanceGasFeePopoverContext();
+  const isAdvancedGasFeeDefault = useSelector(getIsAdvancedGasFeeDefault);
+  const advancedGasFeeValues = useSelector(getAdvancedGasFeeValues);
```

Comment:
`const advancedGasFeeValues = useSelector(getAdvancedGasFeeValues);` looks like not needed

Reply:
With the recent changes, we are curremtly using the advancedGasFeeValues

Reply:
we really do not need these 2 selectors: getAdvancedGasFeeValues, getIsAdvancedGasFeeDefault

`getAdvancedGasFeeValues` should do